layouts/partials/head.html

```diff
@@ -1,4 +1,14 @@
 <head>
+  <!-- Google Tag Manager -->
+  <script>(function (w, d, s, l, i) {
```

Comment:
Small nit: I would expect this script to be closer to the other scripts in the `<head>` tag.

Reply:
I agree, but the AC states the following:

> Paste this code as high in the <head> of the page as possible:

Reply:
I don't see any advantages of putting the script tag right after the head tag, though.  If anything, it makes things look inconsistent.